docs/Buttons.md

```diff
@@ -627,16 +780,24 @@ This button is an internal component used by react-admin in [the Filter button/f
 </video>
 
 
-#### `sx`: CSS API
+### `sx`: CSS API
 
 To override the style of all instances of `<FilterButton>` using the [application-wide style overrides](./AppTheme.md#theming-individual-components), use the `RaFilterButton` key.
 
-## Record Buttons
-
-### `<UpdateButton>`
+## `<UpdateButton>`
```

Comment:
There was a short doc for `UpdateButton` here, and a dedicated page in `UpdateButton.md`. I don't see why that button deserves a specific page. So I moved the content back here, changed the  link in Reference.md to point here, and removed the `UpdateButton` item in the main navigation. 

We must keep the `UpdateButton.md` file though, to avoid 404 on external links.